common/model/tests/test_transaction.c

```diff
@@ -49,12 +49,125 @@ void test_reset(void) {
   transaction_free(transaction);
 }
 
+void test_read_write_transaction_obj(void) {
+  flex_trit_t test_address[FLEX_TRIT_SIZE_243];
+  int64_t test_value = 123456;
+  flex_trit_t test_obsolete_tag[FLEX_TRIT_SIZE_81];
+  uint64_t test_timestamp = 1477037811737;
+  int64_t test_current_index = 2;
+  int64_t test_last_index = 5;
+  flex_trit_t test_bundle[FLEX_TRIT_SIZE_243];
+
+  flex_trit_t test_trunk[FLEX_TRIT_SIZE_243];
+  flex_trit_t test_branch[FLEX_TRIT_SIZE_243];
+  uint64_t test_attachment_timestamp = 1477037821737;
+  uint64_t test_attachment_timestamp_lower = 1477037831737;
+  uint64_t test_attachment_timestamp_upper = 1477037841737;
+  flex_trit_t test_nonce[FLEX_TRIT_SIZE_81];
+  flex_trit_t test_tag[FLEX_TRIT_SIZE_81];
+
+  flex_trit_t test_hash[FLEX_TRIT_SIZE_243];
+  flex_trit_t test_signature_or_message[FLEX_TRIT_SIZE_6561];
+
+  uint64_t test_snapshot_index = 2345;
+  bool test_solid = true;
+  uint64_t test_arrival_timestamp = 1477037812737;
+
+  iota_transaction_t transaction;
+
+  // write transaction values
+  flex_trits_from_trytes(test_address, FLEX_TRIT_SIZE_243, TEST_ADDRESS_0,
```

Comment:
flex trits functions always use size in trits or in trytes, not in flex trits. Please refer to the prototype doccumentation. Here you need to replace `FLEX_TRIT_SIZE_243` with `NUM_TRITS_ADDRESS` and so on...